src/Parser/FireMage/CHANGELOG.js

```diff
@@ -0,0 +1,3 @@
+export default `
+10-14-2017 - Added Spec. (Fyruna)
```

Comment:
Changelog entries prefixed by class/spec, I think so they show right in overall changelog. Also attribution should be "(by Author)" Try  `10-14-2017 - Fire Mage: Added Spec. (by Fyruna)`

Reply:
Disagree - now that the changelogs are split by class there's no need to prefix it (it made sense when it was 1 Changelog for the whole analyzer.